apps/dot_ext/forms.py

```diff
@@ -133,6 +134,13 @@ def clean_logo_image(self):
             validate_logo_image(logo_image)
         return logo_image
 
+    def clean_require_demographic_scopes(self):
+        require_demographic_scopes = self.cleaned_data.get('require_demographic_scopes')
+        if type(require_demographic_scopes) != bool:
```

Comment:
Since `require_demographic_scopes` is a radio button that is either True or False, is there any way this condition gets met?  Does this validate apps that don't have an existing value for this yet perhaps?

Reply:
@njdister Good catch! Existing applications will have this value set to Null after the migrations. I'll update the code to handle that case.